.github/workflows/claude.yml

```
@@ -35,6 +35,9 @@ jobs:
         uses: anthropics/claude-code-action@beta
         with:
           claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)